### ISSUE TITLE
may not support image png conversion base64

### ISSUE THREAD

**Issue by MrZhang123** [Snapp-FidMe/react-native-img-to-base64#4]
I conversion .png to base64 , when i decoding base64 to png ,  the background is black

**Comment by Michaelvilleneuve** [comment]
That's possible, we mainly designed this lib to send camera pictures which are JPEG so we didn't focus on PNG.

I add it to the roadmap, in the meantime feel free to PR :)

**Comment by Desintegrator** [comment]
> That's possible, we mainly designed this lib to send camera pictures which are JPEG so we didn't focus on PNG.
> 
> I add it to the roadmap, in the meantime feel free to PR :)

any news on it?

**Comment by kirantripathi** [comment]
@Desintegrator  did you find any way to remove the dark background in image?

**Comment by Desintegrator** [comment]
> @Desintegrator did you find any way to remove the dark background in image?

@kirantripathi yes, its works correctly in ios without any changes and for android you need to open RNImgToBase64Module.java in lib sources and change single line in "bitmapToBase64" method `bitmap.compress(Bitmap.CompressFormat.JPEG, 80, byteArrayOutputStream);` to `bitmap.compress(Bitmap.CompressFormat.PNG, 80, byteArrayOutputStream);`

**Comment by kirantripathi** [comment]
> > @Desintegrator did you find any way to remove the dark background in image?
> 
> @kirantripathi yes, its works correctly in ios without any changes and for android you need to open RNImgToBase64Module.java in lib sources and change single line in "bitmapToBase64" method `bitmap.compress(Bitmap.CompressFormat.JPEG, 80, byteArrayOutputStream);` to `bitmap.compress(Bitmap.CompressFormat.PNG, 80, byteArrayOutputStream);`

@Desintegrator thank you for your help.
